### PR TITLE
Assignment 1 - Tom-Rohr

### PR DESCRIPTION
**Pull request by Tom-Rohr**

### CODE REVIEW

**Comment by AndrewKorchemniy** [comment]
will review

File: wordle-web/src/App.vue [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<script setup lang="ts">
+import { RouterLink, RouterView } from 'vue-router'
+import NavBar from '@/components/NavBar.vue'
+</script>
+
+<template>
+  <div class="main">
```
Comment (AndrewKorchemniy) [inline wordle-web/src/App.vue:7]: To further build this out in the future and to replace the default vue stuff, I recommend refactoring the main template to utilize the grid system from vuetify. The grid system is shared among nearly all UI libraries, and there are many examples in the vuetify docs. It's an easy way to quickly throw together content layouts.
https://vuetifyjs.com/en/components/grids/

File: wordle-web/src/views/GameView.vue [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    </div>
+  </template>
+  
+  <style>
+  @media (min-width: 1024px) {
+    .game {
+
+    }
+  }
+  </style>
```
Comment (AndrewKorchemniy) [inline wordle-web/src/views/GameView.vue:34]:
```suggestion
```

**Review by AndrewKorchemniy — COMMENTED** [review]
Great work!
I'm not sure if committing from main is ideal. Someone commented in the teams chat about a way to get it running from other branches.

## Assignment 1

The purpose of this assignment is to solidify your learning of:

- Building a Vue 3, Vuetify, TypeScript App
- Unit testing infrastructure using Jest
- Creating a CI/CD pipeline with GitHub Actions
- Deploying the site to Azure

## Instructions

1. Create a Vue app ✅
   - Uses TypeScript ✅
   - Add Vuetify ✅
   - Uses Vitest ✅

3. Create a simple page called game.vue ✅
   - Page should have a card with a heading that with a clever name for the game ✅
   - some default placeholder body text ✅
   - and a button to take you back to the home page ✅
   - there should be a button on the home page to access this page ✅

4. Check all the code into your repo fork ✅

5. Create a static web app in Azure ✅
   - Create the container where your GitHub repo will deploy
   - Link the site with your GitHub repo

6. Update the CI/CD pipeline ✅
   - Pipeline runs on each check in
   - Vitest run and pass ✅
   - Vue app builds ✅
   - Vue app deploys to Azure ✅

7. Azure site works ✅
   - Post your Azure URL in your readme.md file. Replace the URL in the readme with your URL. ✅

## Extra Credit

- Come up with up to three unique ideas for stories we could implement as part of our project. Post these in your readme.md. ❌
- One point per idea up to three points

File: wordle-web/src/views/GameView.vue [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
...voluptates. Reprehenderit sit, ad id quis veritatis odit nostrum minima a officiis ut iusto nobis quod quae beatae cumque consequuntur. Itaque, voluptas atque pariatur id laboriosam eligendi vel repellat maxime delectus incidunt iure mollitia deleniti officia quia nobis officiis doloribus neque natus aperiam nostrum? Voluptate aut quasi alias accusantium dolore saepe ipsam veritatis, repellendus optio quisquam. Necessitatibus iure numquam rem at dolor molestiae laborum veritatis quasi deleniti voluptatem soluta ipsa iusto accusamus, magnam laboriosam reprehenderit tenetur fugit minus explicabo maxime exercitationem nobis est. Eos temporibus asperiores provident consequuntur ullam doloremque! Quia esse non dolor quo laudantium accusamus explicabo libero dolores dolorum, quod consequatur at eveniet consectetur ipsa eaque, veniam id? Nostrum fugit consequatur, ea nihil excepturi aliquam beatae dolores atque tempora illum provident earum deleniti magnam aliquid dolore alias veniam optio blanditiis delectus, sunt accusantium rerum vitae. Corrupti autem libero facere voluptatibus labore alias laborum, repellendus blanditiis veniam officia expedita exercitationem, nisi quas saepe nam?</p>
```
Comment (mmwoodfo) [inline wordle-web/src/views/GameView.vue:13]: No need to wrap this in a p tag
```suggestion
              Lorem ipsum dolor sit amet consectetur adipisicing elit. Esse vitae est quae magni sequi harum, expedita laborum ullam alias perspiciatis. Cupiditate at error laborum nostrum, magni praesentium maxime possimus voluptatum inventore molestias unde sed facere sunt ea dolorem suscipit eum mollitia ad id repellendus dolore voluptates. Reprehenderit sit, ad id quis veritatis odit nostrum minima a officiis ut iusto nobis quod quae beatae cumque consequuntur. Itaque, voluptas atque pariatur id laboriosam eligendi vel repellat maxime delectus incidunt iure mollitia deleniti officia quia nobis officiis doloribus neque natus aperiam nostrum? Voluptate aut quasi alias accusantium dolore saepe ipsam veritatis, repellendus optio quisquam. Necessitatibus iure numquam rem at dolor molestiae laborum veritatis quasi deleniti voluptatem soluta ipsa iusto accusamus, magnam laboriosam reprehenderit tenetur fugit minus explicabo maxime exercitationem nobis est. Eos temporibus asperiores provident consequuntur ullam doloremque! Quia esse non dolor quo laudantium accusamus explicabo libero dolores dolorum, quod consequatur at eveniet consectetur ipsa eaque, veniam id? Nostrum fugit consequatur, ea nihil excepturi aliquam beatae dolores atque tempora illum provident earum deleniti magnam aliquid dolore alias veniam optio blanditiis delectus, sunt accusantium rerum vitae. Corrupti autem libero facere voluptatibus labore alias laborum, repellendus blanditiis veniam officia expedita exercitationem, nisi quas saepe nam?
```

File: wordle-web/src/views/GameView.vue [hunk 4]
```diff
@@ -0,0 +1,34 @@
+<template>
+    <div class="game">
+        <v-card>
+          <v-card-title>
+              <h1>Word War 1</h1>
+          </v-card-title>
+
+          <v-card-subtitle>
+              <p>Title work in progress</p>
```
Comment (mmwoodfo) [inline wordle-web/src/views/GameView.vue:9]: Same here, your v-card-subtitle tag is all you need
```suggestion
              Title work in progress
```

File: wordle-web/src/views/GameView.vue [hunk 5]
```diff
@@ -0,0 +1,34 @@
+<template>
+    <div class="game">
+        <v-card>
+          <v-card-title>
+              <h1>Word War 1</h1>
```
Comment (mmwoodfo) [inline wordle-web/src/views/GameView.vue:5]: No need to wrap this in an h1 tag, if the sizing for your v-card-title isn't to your liking, you can use the typography classes from vuetify.

File: wordle-web/src/views/GameView.vue [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+          </v-card-text>
+          
+          <v-card-actions>
+            <RouterLink to="/">
```
Comment (mmwoodfo) [inline wordle-web/src/views/GameView.vue:17]: No need to wrap a router link around your button. Just place `to="/"` on the v-btn

**Pull request closed without merging**